docs/overview/set-up-your-dev-environment.md

```diff
@@ -34,34 +34,13 @@ The main tools to be installed are:
 - npm
 - A code editor of your choice
 - A project creation tool:
-
-    - Office Yeoman Generator (Yo Office)
-    - Teams Toolkit (requires Visual Studio Code)
-
+  - Office Yeoman Generator (Yo Office)
+  - Teams Toolkit (requires Visual Studio Code)
```

Comment:
Should we put the Dev Kit here? Seems like 3 options to create add-in project.
1. Create with command - Yeoman
2. Create JSON-based add-in with extension - Teams toolkit
3. Create XML-based add-in with extension - Dev Kit (preview)

Reply:
we may also need an update in line21

Reply:
Hmm, since we're making a new "VS Code" tab, I think it actually makes more sense to move the TTK stuff under the VS Code tab.